src/VirtualClient/VirtualClient.Dependencies/MountDisks.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Dependencies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Platform;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// An abstract virtual client action that formats the disks before execution
+    /// </summary>
+    [UnixCompatible]
+    [WindowsCompatible]
+    public class MountDisks : VirtualClientComponent
```

Comment:
The new dependency can ensure a different, user-specified mount point for each volume on the disk.
With multiple benchmarks in the workload, it would be better to store their model specific checkpoints and data in separate volumes and mount them at user specific mount points to avoid the necessity to filter the volumes of the disk to find the right volume to use for that particular benchmark.